salt/utils/gitfs.py

```diff
@@ -2604,6 +2630,29 @@ def fetch_remotes(self, remotes=None):
             name = getattr(repo, "name", None)
             if not remotes or (repo.id, name) in remotes or name in remotes:
                 try:
+                    # Find and place fetch_request file for all the other branches for this repo
+                    for branch in os.listdir(repo.get_cache_hash()):
+                        # Don't place fetch request in current branch being updated
+                        if branch == repo.get_cache_basename():
+                            continue
+                        branch_salt_dir = salt.utils.path.join(
+                            repo.get_cache_hash(), branch, ".git", ".salt"
+                        )
+                        fetch_path = salt.utils.path.join(
+                            branch_salt_dir, "fetch_request"
+                        )
+                        if os.path.isdir(branch_salt_dir):
+                            try:
+                                with salt.utils.files.fopen(fetch_path, "w"):
+                                    pass
+                            except Exception as exc:  # pylint: disable=broad-except
```

Comment:
```suggestion
                            except Exception as exc:  # pylint: disable=broad-except
```
Do we really need this broad exception?